jenkins/automation/github-release.groovy

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+pipeline {
+
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+
+    parameters {
+        string(name: 'GIT_TAG', defaultValue: '', description: 'Image tag/Commit tag required for GitHub Release')
+        string(name: 'SERVICES_VERSION', defaultValue: '', description: 'Services(cortx-k8s) version on which image deployment is tested')
+        string(name: 'CHANGESET_URL', defaultValue: '', description: 'CHNAGESET.md file url.')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for GitHub Release script', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for GitHub release script', trim: true)
+    }
+
+    stages {
+        stage('Checkout Script') {
+            steps {
+                script { build_stage = env.STAGE_NAME }             
+                script {
+                    checkout([$class: 'GitSCM', branches: [[name: "${CORTX_RE_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "${CORTX_RE_REPO}"]]])
+                }
+            }
+        }
+        
+        stage("Create GitHub Release") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                    withCredentials([string(credentialsId: 'gaurav-github-token', variable: 'GH_TOKEN')]) {
+                        env.github_release_url = sh( script: """
+                            bash scripts/release_support/github-release.sh -t $GIT_TAG -v $SERVICES_VERSION -c $CHANGESET_URL
+                        """, returnStdout: true).trim()
```

Comment:
`returnStdout: true` - Any specific reason to add this.

Reply:
I guess the stage will not exit out even if the exit code is non-zero if `returnStdout: true` . This may give false status.

Reply:
but we are expecting output to get stored in variable here. it will not give a value for a variable